src/web/Link.tsx

```diff
@@ -47,11 +51,11 @@ export class Link extends React.Component<Types.LinkProps, {}> {
     private _longPressTimer: number;
 
     render() {
-       // SECURITY WARNING:
-       //   Note the use of rel='noreferrer'
-       //   Destroy the back-link to this window. Otherwise the (untrusted) URL we are about to load can redirect OUR window.
-       //   See: https://mathiasbynens.github.io/rel-noopener/
-        return (
+        // SECURITY WARNING:
+        //   Note the use of rel='noreferrer'
+        //   Destroy the back-link to this window. Otherwise the (untrusted) URL we are about to load can redirect OUR window.
+        //   See: https://mathiasbynens.github.io/rel-noopener/
+        return this.props.url ? (
```

Comment:
Is there really a need for RX.Link to support the case where there's no provided URL? Why not just make 'url' a required prop?

Reply:
Yes, valid point. I should have addressed it from the other direction.